lisa/search_space.py

```diff
@@ -710,3 +710,15 @@ def create_set_space(
     else:
         set_space = None
     return set_space
+
+
+def decode_empty_count_space(
+    data: Any, base_type: Any, allowed_values: Any, is_allow_set: bool = False
```

Comment:
`allowed_values` should be `default_values`

Reply:
updated